src/containers/AccountPageContainer.jsx

```diff
@@ -1,28 +1,6 @@
-import React, { PropTypes } from 'react';
```

Comment:
I think you still need to import React in here, but I could be wrong. Have you tested this?